app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find params[:id]
```

Comment:
em dùng `find_by` thay cho `find` nhé.
2 method này khác nhau ntn nhỉ?

Reply:
theo em hiểu thì find dựa vào id là chính để tìm kiếm record, xong mới đến các condition khác
còn find_by thì tìm theo các cặp key-value
find trả về exception nếu id không tồn tại hoặc vượt quá số record dễ gây crash còn find_by trả về nil nếu không tìm thấy nên an toàn hơn